OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1789,6 +1789,9 @@ public void TestUnsupportedVariableType()
             var featureVariableStringRandomType = Optimizely.GetFeatureVariableString("", "any_key", TestUserId);
             Assert.IsNull(featureVariableStringRandomType);
 
+            var featureVariableStringJsonType = Optimizely.GetFeatureVariableString("unsupported_variabletype", "string_json_key", TestUserId);
```

Comment:
what's the point to add test here?

Reply:
It was an old test which was testing that getFeatureVariableString will return string value  even when given json as subtype but this is no longer the case here. So now if the sub type is JSON and we will call getFeatureVariableString to get that variable, it will return null because type will not be matched.